### PR TITLE
Updated github actions

### PR DESCRIPTION
**Pull request by hussnain612**
Updated variable naming to make it more generic
Removed publish to DockerHub

### CODE REVIEW

File: .github/workflows/operator_pull_request.yaml [hunk 1]
```diff
@@ -73,17 +69,17 @@ jobs:
...
+    - name: Check if registry secrets are set
       run: |
-        if [ "${{ secrets.STAKATER_NEXUS_REGISTRY }}" == "" ] || [ "${{ secrets.STAKATER_NEXUS_USERNAME }}" == "" ] || [ "${{ secrets.STAKATER_NEXUS_PASSWORD }}" == "" ]; then
-          echo "Required Secrets 'STAKATER_NEXUS_REGISTRY' or 'STAKATER_NEXUS_USERNAME' or 'STAKATER_NEXUS_PASSWORD' are not set!"
+        if [ "${{ secrets.REGISTRY_URL }}" == "" ] || [ "${{ secrets.REGISTRY_USERNAME }}" == "" ] || [ "${{ secrets.REGISTRY_PASSWORD }}" == "" ]; then
+          echo "Required Secrets 'REGISTRY_URL' or 'REGISTRY_USERNAME' or 'REGISTRY_PASSWORD' are not set!"
           exit 1
         fi
 
     - name: Check if registry secrets are set
```
Comment (MuneebAijaz) [inline .github/workflows/operator_pull_request.yaml:79]: shouldn't it be slack secret?

Comment (hussnain612) [reply .github/workflows/operator_pull_request.yaml]: Updated

File: .github/workflows/operator_push.yaml [hunk 2]
```diff
@@ -14,72 +14,45 @@ on:
...
-        required: true
-
-      STAKATER_DELIVERY_SLACK_WEBHOOK:
+      SLACK_WEBHOOK_URL:
         description: "Secret to send success/failure message to slack"
         required: true
 
-      STAKATER_NEXUS_REGISTRY:
-        description: "Nexus docker registry to publish image"
+      CONTAINER_REGISTRY_URL:
```
[the remark this answers is not in the document]
Comment (hussnain612) [reply .github/workflows/operator_push.yaml:34]: Had to cater for both docker container registry and helm chart registry

[the remark this answers is not in the document]
Comment (MuneebAijaz) [reply .github/workflows/operator_push.yaml:34]: better to use same convention in above file then

[the remark this answers is not in the document]
Comment (hussnain612) [reply .github/workflows/operator_push.yaml:34]: Updated

**Review by MuneebAijaz — APPROVED** [review]

**Pull request merged**